src/validators/literal.rs

```diff
@@ -75,6 +97,17 @@ impl<T: Clone + Debug> LiteralLookup<T> {
         input: &'data I,
     ) -> ValResult<'data, Option<(&'data I, &T)>> {
         // dbg!(input.to_object(py).as_ref(py).repr().unwrap());
```

Comment:
Kill this old `dbg!` while you're at it?

Reply:
```suggestion
```